COVID19-Event-Risk-Planner/server/risk-game-reactivity.R

```diff
@@ -1,76 +1,81 @@
-  if (input$`nav-page` == "game" && is.null(input$cookies$consent)){
+  if (input$`nav-page` == "game" && is.null(input$cookies$consent)) {
     inputSweetAlert(
-      session = session, inputId = "game_consent",
+      session = session,
+      inputId = "game_consent",
       inputPlaceholder = "I am over 18 and in the US",
-      title = "Are you over 18 and in the US?", text = paste0(
-        "Users under 18 and/or who reside outside the US",
+      title = "Are you over 18 and in the US?",
+      text = paste0(
+        "Users under 18 and/or those who reside outside the US",
         " are encouraged to use the risk prediction tools, ",
-        "but unfortunately we cannot save your survey feedback.  ",
+        "unfortunately we cannot save your survey feedback.  ",
         "Select 'No' if you are not eligible or would like to ",
         "opt out of having your responses saved for research purposes.  ",
-        "Please see the About page for more details"),
-      type = "question", reset_input = TRUE, btn_labels = "Confirm", input="radio",
-      inputOptions = c("yes" = "Yes", "no" = "No"), inputValue = "yes"
+        "Please see the About page for more details"
+      ),
+      type = "question",
+      reset_input = TRUE,
+      btn_labels = "Confirm",
+      input = "radio",
+      inputOptions = c("yes" = "Yes", "no" = "No"),
+      inputValue = "yes"
     )
   }
 })
 
-observeEvent(input$game_consent,{
-  if (input$game_consent == "yes"){
-    session$sendCustomMessage("cookie-set", list(
-      name = "consent", value = "yes"
-    ))
+observeEvent(input$game_consent, {
+  if (input$game_consent == "yes") {
+    session$sendCustomMessage("cookie-set", list(name = "consent", value = "yes"))
   } else {
-    session$sendCustomMessage("cookie-set", list(
-      name = "consent", value = "no"
-    ))
+    session$sendCustomMessage("cookie-set", list(name = "consent", value = "no"))
   }
 })
 
 observeEvent(input$risk_state, {
-
-  if (length(geo_county()) > 0){
+  if (length(geo_county()) > 0) {
     geo_county(NULL)
   } else{
-    updateSelectizeInput(
-      "risk_county",
-      choices = usa_counties %>% filter(stname == !!input$risk_state) %>% pull(NAME) %>% sort %>% unique(),
-      session = session
-    )
+    if (input$risk_state == "USA") {
+      game_counties = "Entire US"
+    } else {
+      game_counties = usa_counties %>% filter(stname == !!input$risk_state) %>% pull(NAME) %>% sort %>% unique()
+    }
+
+    updateSelectizeInput("risk_county",
+                         choices = game_counties,
+                         session = session)
   }
 }, ignoreNULL = T)
 
 output$location_selector <- renderUI({
   if (input$geolocation) {
     HTML("<p class='loc-text success'>Detected your location automatically</p>")
-    # input$geolocation <- FALSE
   }
   else{
-    HTML("<p class='loc-text'><b>Please choose your location below</b></p>")
+    HTML("<p class='loc-text'>Please choose your location below</p>")
```

Comment:
I prefer this unbolded, double check with @sjbeckett

Reply:
I think users may actually see this section last on the page (due to sizing of other elements), yet we would like for them to interact with it first. 

I think making this text larger would help - probably as large at the quiz question text. Bold/color may also help it stand out. What do you think?